viadot/tasks/sap_rfc.py

```diff
@@ -97,6 +110,9 @@ def run(
             multiple options are automatically tried. Defaults to None.
             replacement (str, optional): In case of sep is on a columns, set up a new character to replace
                 inside the string to avoid flow breakdowns. Defaults to "-".
+            credentials (dict, optional): The credentials to use to authenticate with SAP.
```

Comment:
```suggestion
            credentials (dict, optional): The credentials to use to authenticate with SAP. Defaults to None.
```

Reply:
Changed docstring to the old one from flow. And variable name from 'credentials' to 'sap_credentials'